### ISSUE TITLE
Ability to make plots comparable

### ISSUE THREAD

**Issue by ElDeveloper** [PitchInteractiveInc/Phinch#41]
Currently there's not a good way to make two different plots comparable, because for each plot you make you get a different coloring scheme (i. e. each taxa is associated with a different color) and thus when putting together two or more of them you need to either manually fix the colors or provide two groups of color-taxa legends.

**Comment by hollybik** [comment]
Do you have a specific example of two plots where you found the colors for the same taxon to be different? I had assumed that the coloring had remained consistent between visualizations, but then again I hadn't specifically investigated this across plots. Note that we're only recycling about ~16 colors in each visualization - we found that using more than that made the color gradients too hard to interpret.

**Comment by ElDeveloper** [comment]
I agree with limiting the number of colors.

The screenshots below show how Firmicutes has two different colors:

![screen shot 2015-01-13 at 11 12 13 am copy](https://cloud.githubusercontent.com/assets/375307/5727100/62f6abd2-9b15-11e4-82a0-2a5b1d8feaad.png)

![screen shot 2015-01-13 at 11 12 04 am copy](https://cloud.githubusercontent.com/assets/375307/5727104/6a21baa0-9b15-11e4-8174-2df547677f77.png)

**Comment by hollybik** [comment]
Thanks for those - are these two screenshots are from two different BIOM files that you independently uploaded? Right now the colors are re-assigned for each BIOM file you uploaded (based on the taxonomy/abundance information per file) - so if your rank abundance info for OTUs varied slightly between files, then the color assignments would differ. Definitely agree that we should fix this to avoid the need for separate keys.

**Comment by ElDeveloper** [comment]
Yes these are two independently uploaded tables. I agree, either that or
being able to order the samples by different categories, for example by
subject and then by timepoint so you end up with something like:

SA-1 SA-2 SA-3 SA-4 | SB-1 SB-2 SB-5 | SC-1 SC-10

Where you have two categories in the metadata HOST_SUBJECT_ID and
Time-point.
